### ISSUE TITLE
appsettings not respecting the environment

### ISSUE THREAD

**Issue by daniloak** [LoginRadius/dot-net-sdk#10]
it would be nice to have the appsettings as the conventions, we usually have appsettings.{Environment}.json, it seems the SDK is always getting appsettings.json only, it would be nice to have this differentiation as the configuration document below 

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1

Suggestion

```
        public SDKConfigHandler()
        {
            var environment = Environment.GetEnvironmentVariable(BaseConstants.ASPNETCORE_ENVIRONMENT);

            var builder = new ConfigurationBuilder()
                .SetBasePath(Directory.GetCurrentDirectory())
                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
                .AddJsonFile($"appsettings.{environment}.json", optional: true, reloadOnChange: true)
                .AddEnvironmentVariables();

            Configuration = builder.Build();
        }
```

**Comment by indrasen715** [comment]
Hi  @daniloak ,

Thanks for bringing this to our notice and we appreciate your valuable feedback on this.
I will review this feedback with the team, and keep you posted on the progress.

Regards
Indrasen

**Comment by indrasen715** [comment]
Hi @daniloak,

Thanks for your contribution, this particular request is included in our latest release version 11.2.0 

https://github.com/LoginRadius/dot-net-sdk/releases/tag/11.2.0

Please feel free to share with us, if you see any other areas of improvement. 

Regards
Indrasen
